timemachine/cpp/src/nonbonded_pair_list.cu

```diff
@@ -59,13 +59,15 @@ void NonbondedPairList<RealType, Negated>::execute_device(
     unsigned long long *d_u,
     cudaStream_t stream) {
 
-    const int tpb = default_threads_per_block;
-    const int num_blocks_pairs = ceil_divide(M_, tpb);
+    if (M_ > 0) {
```

Comment:
Without this, there is a failure to launch the kernel in the test. Probably an uncommon case